test/test-mongocrypt-crypto-hooks.c

```diff
@@ -80,7 +80,7 @@ _aes_256_cbc_encrypt (void *ctx,
    bson_string_append_printf (call_history, "ret:%s\n", BSON_FUNC);
    if (0 == strcmp ((char *) ctx, "error_on:aes_256_cbc_encrypt")) {
       mongocrypt_status_set (
-         status, MONGOCRYPT_STATUS_ERROR_CLIENT, 1, "error message", -1);
+         status, MONGOCRYPT_STATUS_ERROR_CLIENT, 1, (char *) ctx, -1);
```

Comment:
This now checks that the error message returned corresponds to the hook. The error message will contain a message specific to that hook. In this case it is: `error_on:aes_256_cbc_encrypt`.

Reply:
Nice!